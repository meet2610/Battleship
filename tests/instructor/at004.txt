bomb([A,4],[D,4])

