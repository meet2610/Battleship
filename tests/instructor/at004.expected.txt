  state 0 OK -> Start a new game
->bomb([A, 4],[D, 4])
  state 1 Game not started -> Start a new game
